tests/unit/shared/test_cmd_proc.py

```diff
@@ -0,0 +1,15 @@
+from vic import lib as vic_lib, VIC_DRIVER
```

Comment:
easier to read if you split these out
```python
from vic import lib as vic_lib
from vic import VIC_DRIVER
```
`PEP8` says it's OK to put imports from the same package on one line, e.g.:
```python
from subprocess import Popen, PIPE
```
but in this case the `as` makes this confusing to read. Alternatively, change the order
```python
from vic import VIC_DRIVER, lib as vic_lib
```